src/modules/endpoint/sagas/watchContentLanguageSwitch.js

```diff
@@ -23,6 +24,10 @@ function * switchContentLanguage (dataContainer: DataContainer, action: SwitchCo
     }
     yield put(setContentLanguage)
 
+    const city = yield select((state: StateType) => state.cityContent && state.cityContent.city)
```

Comment:
Why don't you just pass `city` as action param?

Reply:
Well, it's not necessary to have a city; actions should be as atomic as they can be imo...

Reply:
I don't get that... what is the difference of passing `city: ?string` as action param or reading it from the state by `select`? The  docs regarding `select` are saying the following: *A saga should preferably depend only on its own internal control state when possible* and I kind of agree with that here because I don't really see the need why we need to do this here. If there is a reason to do this here, it probably makes sense in our other sagas as well.

Reply:
I'm also for passing this as a param if possible. It is way more difficult to tell which value `cityContent.city` has at that exact point because it depends on previously `putted` actions.

Reply:
Keeping functions and sagas pure (as they tell in the docu) is generally a good style.

Reply:
Yeah, I might change it back... I'm still a little bit struggling with the asynchronous nature of sagas though (in theory it could happen that the `city` in the state is different to when the saga was fired (due to actions that dispatched while the saga was running), which might lead to other problems, if the saga dispatches further actions to the state assuming a wrong `city` 🤔

Reply:
Changed it back